samples/apps/bookinfo/src/mongodb/ratings_data.json

```diff
@@ -1,2 +1,2 @@
+{rating: 6}
 {rating: 5}
-{rating: 4}
```

Comment:
maybe 1 and 4 for a 5 total to make it different from the hardcoded one (I forgot what the hardcoded values are)

Reply:
Oh shoot. This change should not be there. It should be 5, 4